packages/e2e-tests/specs/block-transforms.test.js

```diff
@@ -168,5 +173,19 @@ maybeDescribe( 'Block transforms', () => {
 				).toMatchSnapshot();
 			}
 		);
+
+		// Test one of the Group transforms separately to avoid creating unecessary tests
+		it( 'correctly transforms one of the available blocks into a group block', async () => {
```

Comment:
I guess if we add tests or change their order a group that was the first may stop being the first and this test may start to fail as the snapshot value becomes unexpected.
As an alternative I think we can apply the group transforms to specific fixtures e.g: paragraph and image:
```
const testTableWithSomeGroupsFiltered = testTable.filter( ( [ originalBlock, fixture, destinationBlock ] ) => ( destinationBlock !== 'Group' || fixture === 'core__paragraph__align-right' || fixture === 'core__image' ) );
```

Reply:
That's a good point. Thanks for raising. I'll fix that now.